tfx/components/testdata/external/README.md

```diff
@@ -13,3 +13,42 @@ df.to_parquet('data.parquet', index=False)
 
 Avro data file can be generated by running the script in
 `avro/generateAvroFile.py`
+
```

Comment:
line 1 "Note that the testing files here are same as [taxi data]" is not true after adding this sequence file, update it